addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -145,6 +145,11 @@ export class OdooEditor extends EventTarget {
                 isRootEditable: true,
                 defaultLinkAttributes: {},
                 getContentEditableAreas: () => [],
+                getPowerboxElement: () => {
+                    const selection = document.getSelection();
+                    if (!selection.isCollapsed || !selection.rangeCount) return;
+                    return closestElement(selection.anchorNode, 'P, DIV');
```

Comment:
This function retrieve an element into which the powerbox is allowed to be triggered.
The H1, ... are themselves contained at least into a DIV so that's fine.
If that DIV is has an attribute `oe-model` in the context of the editor, no elements will be returned (as the powerbox is not allowed there).

Reply:
I have made the change to make the `if` "positive".

Reply:
> The H1, ... are themselves contained at least into a DIV so that's fine.

The DIV being the editable in many cases? Then I guess my confusion is why the "P" in the selector in that case. What makes a "P" relevant and a "H1" not?

Reply:
Because we want the closet between a P and a DIV in order to use it in `_makeHint`. If the DIV would be provided, it would not satisfy the condition inside `_makeHint` (`block && block.innerHTML.trim();`).

The difference with the H1, ... is that for theses elements, the hints are always active regardless of the current selection.

For the hint `Type "/" for commands`, it is only active when the anchorNode is within a P or DIV.